package.json

```
@@ -100,10 +100,10 @@
   "peerDependencies": {
     "@storybook/svelte": "^7.0.0",
     "@storybook/theming": "^7.0.0",
-    "@sveltejs/vite-plugin-svelte": "^2.0.0",
+    "@sveltejs/vite-plugin-svelte": "^3.0.0",
```

If no other changes are needed it looks like we can safely support both v2 and v3?
Only having v3 here won't work, as that only supports Vite 5, so even if our Vite peer deps allows 4+5, in the end the package manager will resolve it to only support Vite 5.
```suggestion
    "@sveltejs/vite-plugin-svelte": "^2.0.0 || ^3.0.0",
```